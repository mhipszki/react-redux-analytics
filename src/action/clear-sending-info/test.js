import clearSendingInfo from './';

test('generates action to clear sending related info in store', () => {
  expect(clearSendingInfo()).toMatchSnapshot();
});
