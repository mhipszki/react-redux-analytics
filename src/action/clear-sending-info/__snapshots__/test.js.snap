// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates action to clear sending related info in store 1`] = `
Object {
  "type": "CLEAR_SENDING_INFO",
}
`;
